grafana_objects/client_grafana_objects_delete_by_uid_test.go

```diff
@@ -0,0 +1,71 @@
+package grafana_objects_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"path"
+	"testing"
+
+	"github.com/logzio/logzio_terraform_client/grafana_objects"
+	"github.com/stretchr/testify/assert"
+)
+
+func deleteMockHandler(t *testing.T) func(http.ResponseWriter, *http.Request) {
```

Comment:
Please refactor the test like in here:
https://github.com/logzio/logzio_terraform_client/blob/master/archive_logs/archive_logs_setup_test.go

Also, please add tests for responses 500, 404 from the server (you can use the archive logs dir as an example for how to do that).

Reply:
done